yb-voyager/src/query/sqltransformer/sqlfiletransformer.go

```diff
@@ -199,10 +202,11 @@ func (t *TableFileTransformer) Transform(file string) (string, error) {
 	}
 
 	if t.shouldAddHashSplitting() {
```

Comment:
let's change the name of this func to be something along the lines of `shouldConfigureShardingStrategyForPK` ?